profile/src/main/java/org/apache/karaf/profile/assembly/Builder.java

```diff
@@ -1655,7 +1655,7 @@ private void installStage(Profile installedProfile, Set<Feature> allBootFeatures
                 }
             }
 
-            new ConfigInstaller(etcDirectory, pidsToExtract)
+            new ConfigInstaller(homeDirectory, etcDirectory, pidsToExtract)
```

Comment:
Why do we need the `homeDirectory` here ? Config should go in `etc`. And it seems to  be the `baseDirectory` (not home).

Reply:
The [documentation](https://karaf.apache.org/manual/latest/#_feature_configuration_files) says it is relative.
> If not specified, the location is relative from the KARAF_BASE variable. It’s also possible to use variable like ${karaf.home}, ${karaf.base}, ${karaf.etc}, or even system properties.

Each location can occur or it is relative to KARAF_BASE. KARAF_HOME is indeed not good. But is home not always equal to base in the AssemblyMojo? Here are no other instances possible? The builder name it home, but it is the custom distribution directory in the maven target directory. What do you think is best?